nservicebus/upgrades/5to6/distributor.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Deprecated Distributor in NServiceBus Version 6
+tags:
+ - upgrade
+ - migration
+related:
+ - samples/scaleout/distributor-upgrade
+---
+
+The [distributor](/nservicebus/scalability-and-ha/distributor) has been deprecated and replaced by [sender-side distribution](/nservicebus/msmq/scalability-and-ha/sender-side-distribution.md) with NServiceBus Version 6.
+
+## Client-side distribution
+
+With sender-side distribution, senders are aware of scaled out receivers, allowing senders to distribute messages directly to the receivers instead of using an intermediary distributor.
+
+
+## Upgrade scenario
+
+This process aims to allow upgrade in production environments without missing events and minimal downtime.
+
+WARNING: To avoid duplicate events, the Distributor node needs to be upgraded to a worker node using NServiceBus 6 (it can be decommissioned after all workers are upgraded to Version 6).
+
+* Upgrade all endpoints communicating to the Distributor (by commands or event subscription messages) to NServiceBus Version 6 first.
+* Shut down the Distributor instance.
+* [Upgrade the Distributor to NServiceBus Version 6](#upgrade-endpoint-to-version-6). This will migrate the Distributor endpoint to a regular worker instance.
+* Start the instance again.
+* [Upgrade the remaining worker nodes to NServiceBus Version 6](#upgrade-endpoint-to-version-6).
+* Once all instances are upgraded to Version 6, configure all endpoints sending commands or receiving events from the Distributor to use [sender-side distribution](/nservicebus/msmq/scalability-and-ha/sender-side-distribution.md) to send messages directly to the scaled out instances.
+
+
+## Upgrade endpoint to Version 6
```

Comment:
as far as I understood, worker nodes also require the distributor package to enlist with a distributor, so this should be valid for all endpoints in a distributor setup.